 pyproject.toml

```
@@ -0,0 +1,69 @@
+[tool.black]
+line-length = 100
+target-version = ['py39']
+skip-string-normalization = true
+
+
+[build-system]
+requires = [
+    "setuptools>=61",
+    "setuptools-scm"
+]
+build-backend = "setuptools.build_meta"
+
+
+[project]
+name = "xarray-schema"
+description = "Schema validation for Xarray objects"
+readme = "README.md"
+license = {text = "MIT"}
+authors = [{name = "Xarray-schema Developers", email = "joe@carbonplan.org"}]
+requires-python = ">=3.8"
```

```suggestion
requires-python = ">=3.9"
```

let's drop 3.8 as part of this PR